### PR TITLE
Task 4 review list

### PR DESCRIPTION
**Pull request by Cnmoosavinia**
Completed Task 4 with a basic layout of all the reviews. Intention is to have the user and title of the review at the top with an image below. As I move onto ticket 5 I will implement an onclick where you can review individual reviews with all details and comments shown there

### CODE REVIEW

File: src/components/review-board.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.review-list {
+  margin: 5px;
+  list-style-type: none;
+  display: grid;
+  grid-gap: 50px;
+  grid-template-columns: repeat(auto-fill, 500px);
```
Comment (callumu21) [inline src/components/review-board.css:6]: Good use of grid, and I can see you're thinking about different screen widths, but don't forget to consider how other UI elements and positions may need to change for desktop. You don't need to do this just yet, but something to think about for future tickets.

File: src/components/review-list.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+function ReviewList({ reviews, setReviews }) {
+  useEffect(() => {
+    getReviewBoard().then((data) => {
+      setReviews(data);
+    });
+  }, []);
+  return (
+    <ul className="review-list">
+      {reviews.map((review) => {
```
Comment (callumu21) [inline src/components/review-list.jsx:13]: Could you give the user some feedback that the page is loading if the fetch request takes longer than expected?

**Review by callumu21 — APPROVED** [review]
Good start! A few minor points:

1. File names for components are usually uppercase.
2. Don't forget to consider mobile first styling, especially for future tickets.
3. It's good practice to give users some form of visual feedback if a page could be slow to load or depend on external data fetches.

If you could incorporate number 3 before merging, that would be great. Otherwise, PR approved 👍

File: src/components/review-list.jsx (hunk 2 above)
Comment (Cnmoosavinia) [reply src/components/review-list.jsx:13]: added a loading state starting as true, then set to false after data has loaded on the screen. in css, shows a loading <p> when loading state is true which is replaced with the grid when rendered

**Pull request merged**